bofire/domain/features.py

```diff
@@ -435,6 +440,26 @@ def sample(self, n: int) -> pd.Series:
         """
         return pd.Series(name=self.key, data=np.random.choice(self.values, n))
 
+    def to_discrete_values(self, values: pd.DataFrame) -> pd.Series:
```

Comment:
```suggestion
    def from_continuous(self, values: pd.DataFrame) -> pd.Series:
```

This fits better in the whole story here, as we also have in the other cases `from` methods to transform back into the valid settings.